benchmarks/performance_test.py

```diff
@@ -1,17 +1,78 @@
 from urlparse4 import urlsplit, urljoin
 from timeit import default_timer as timer
 
-total = 0
+from sys import argv
 
-with open('urls/chromiumUrls.txt') as f:
-    for url in f:
+encode = False
 
-        start = timer()
+try:
+    if argv[1] == "encode":
```

Comment:
Maybe use ``argparse`` library instead? It's in stdlib both for python 2 and 3, and will make the code much nicer